doc/distrib/NodeHelpFiles/_review/Revit.Application.Document.PurgeUnused.md

```diff
@@ -0,0 +1,8 @@
+## In Depth
+`Document.PurgeUnused` purges elements that are not used from the current Revit document. `Document.PurgeUnused` includes a "Full Purge" option which mimics performing a purge unused in the Revit UI multiple times. 
+
+In the example below, all unused elements are purged from the current Revit document.
```

Comment:
What do the integers in the output represent?

Reply:
The `ElementId` of elements purged.